bip-0372.mediawiki

```diff
@@ -115,17 +115,17 @@ after <tt>PSBT_IN_PARTIAL_SIG</tt> is constructed.
 
 The scope of this proposal is deliberately kept narrow; it addresses
 only spending of transaction outputs containing P2C tweaks - and does not
-addresses construction of a new P2C commitments or transactions containing them
+address construction of new P2C commitments or transactions containing them
 in their outputs.<ref>'''Why only spending of P2C tweaked outputs is covered'''
-P2C tweaks commit to external data, some of which may represent certain value
-(like in some sidechains, single-use-seal applications like RGB etc). Creation
-of such outputs much allow hardware devices to understand the structure of such
+P2C tweaks commit to external data, some of which may represent certain values
+(like in some sidechains, single-use-seal applications like RGB, etc). Creation
+of such outputs may allow hardware devices to understand the structure of such
```

Comment:
This is a bit clunky before and after. I suspect "much allow" was supposed to be "must allow", which would be more readable as "would require".

Reply:
Thanks for taking a look. 
@dr-orlovsky what is your intended meaning here?

Reply:
Happy to be wrong, thanks for chiming in!